bin/visualqa.py

```diff
@@ -318,7 +336,19 @@ def plot_train_metrics(train_stats, options, plot_type='epochs'):
 
 
 def train(model, dataset, options, val_dataset=None, attention_model=None):
-
+    """
+        Trains the Keras model.
```

Comment:
we should also mention that it saves the models and metrics

Reply:
done